src/SprykerEco/Zed/Heidelpay/HeidelpayConfig.php

```diff
@@ -135,6 +144,30 @@ public function getEncryptionKey(): string
         return $this->get(HeidelpayConstants::CONFIG_ENCRYPTION_KEY);
     }
 
+    /**
+     * @return string
+     */
+    public function getEasycreditCriteriaRejectedDeliveryAddress(): string
+    {
+        return $this->get(HeidelpayConstants::CONFIG_HEIDELPAY_EASYCREDIT_CRITERIA_REJECTED_DELIVERY_ADDRESS);
+    }
+
+    /**
+     * @return float
+     */
+    public function getEasycreditCriteriaGrandTotalMoreThan(): float
+    {
+        return $this->get(HeidelpayConstants::CONFIG_HEIDELPAY_EASYCREDIT_CRITERIA_GRAND_TOTAL_MORE_THAN);
+    }
+
+    /**
+     * @return float
+     */
+    public function getEasycreditCriteriaGrandTotalLessThan(): float
```

Comment:
I guess method and const name should be something like `getEasycreditCriteriaGrandTotalMaxLimit`.